### PR TITLE
Add post: Beta location analysis - School placements

### PR DESCRIPTION
**Pull request by simonwhatley**
The posts are:

- [Beta location analysis - phase 1](https://deploy-preview-1327--bat-design-history.netlify.app/manage-school-placements/beta-location-analysis-phase-1/)
- [Beta location analysis - phase 2 - representativeness](https://deploy-preview-1327--bat-design-history.netlify.app/manage-school-placements/beta-location-analysis-phase-2/)
- [Beta location analysis - phase 3 - expansion of the sample area](https://deploy-preview-1327--bat-design-history.netlify.app/manage-school-placements/beta-location-analysis-phase-3/)
- [Beta location analysis - phase 4 - understanding Leeds](https://deploy-preview-1327--bat-design-history.netlify.app/manage-school-placements/beta-location-analysis-phase-4/)

### CODE REVIEW

**Comment by simonwhatley** [comment]
[Beta Location Analysis - Design History](https://trello.com/c/SFCmsHBr/334-beta-location-analysis-design-history)

File: app/posts/manage-school-placements/2024-02-23-beta-location-analysis-phase-3.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+## Birmingham-based providers expanded network
+
+In general, providers in Birmingham operate across a wide area. This will be partially influenced by how we have classified the location of SCITTs, as some regional providers have the most placement school partners in Birmingham.
+
+Upon investigating the potential of these areas to improve Birmingham's representativeness, we chose Sandwell as the most appropriate region. Staffordshire placement schools had similar characteristics to Birmingham.
+
+![Heatmap showing the number of placement schools in each local authority that hosted trainees from a provider located in Birmingham. The lighter the colour, the greater the number of placement schools.](birmingham-based-providers-expanded-network.png "Heatmap showing the number of placement schools in each local authority that hosted trainees from a provider located in Birmingham. The lighter the colour, the greater the number of placement schools.")
+
+| Local authority | Number of providers in the area | Number of placements schools in the area |
```
Comment (Arjun-Kharbanda) [inline app/posts/manage-school-placements/2024-02-23-beta-location-analysis-phase-3.md:57]: I think 'Number of providers in the area' could confuse readers, could we use something like 'Number of Birmingham based providers operating in the area'

File: app/posts/manage-school-placements/2024-02-23-beta-location-analysis-phase-3.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+## Leeds-based providers expanded network
+
+All the providers in Leeds also operate in Bradford, Calderdale, and Kirklees, so these are both excellent options for a second region.
+
+Bradford has many placement schools that are ‘Academies’, so this region has the best potential to improve the representativeness of the Leeds Placement schools.
+
+![Heatmap showing the number of placement schools in each local authority that hosted trainees from a provider located in Leeds. The lighter the colour, the greater the number of placement schools.](leeds-based-providers-expanded-network.png "Heatmap showing the number of placement schools in each local authority that hosted trainees from a provider located in Leeds. The lighter the colour, the greater the number of placement schools.")
+
+| Local authority | Number of providers in the area | Number of placements schools in the area |
```
Comment (Arjun-Kharbanda) [inline app/posts/manage-school-placements/2024-02-23-beta-location-analysis-phase-3.md:91]: Same as previous comment but for Leeds based providers

File: app/posts/manage-school-placements/2024-02-23-beta-location-analysis-phase-3.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+
+## Tyneside-based providers expanded network
+
+6 of the providers in Tyneside also operate in Northumberland and Durham.
+
+Both have similar characteristics, although Northumberland has a better distribution of rural-urban classification.
+
+![Heatmap showing the number of placement schools in each local authority that hosted trainees from a provider located in Tyneside. The lighter the colour, the greater the number of placement schools.](tyneside-based-providers-expanded-network.png "Heatmap showing the number of placement schools in each local authority that hosted trainees from a provider located in Tyneside. The lighter the colour, the greater the number of placement schools.")
+
+| Local authority | Number of providers in the area | Number of placements schools in the area |
```
Comment (Arjun-Kharbanda) [inline app/posts/manage-school-placements/2024-02-23-beta-location-analysis-phase-3.md:128]: Tyneside based providers

File: app/posts/manage-school-placements/2024-02-23-beta-location-analysis-phase-4.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+    - text: Beta location analysis - phase 1
+      href: /manage-school-placements/beta-location-analysis-phase-1/
+    - text: Beta location analysis - phase 2
+      href: /manage-school-placements/beta-location-analysis-phase-2/
+    - text: Beta location analysis - phase 3
+      href: /manage-school-placements/beta-location-analysis-phase-3/
+---
+
+
+
```
Comment (Arjun-Kharbanda) [inline app/posts/manage-school-placements/2024-02-23-beta-location-analysis-phase-4.md:21]: Are providers alright with us sharing this information as they are named here - In other tables they are part of aggregate statistics so this is less of a concern?

**Review by Arjun-Kharbanda — APPROVED** [review]
lgtm!

File: app/posts/manage-school-placements/2024-02-23-beta-location-analysis-phase-3.md (hunk 1 above)
Comment (simonwhatley) [reply app/posts/manage-school-placements/2024-02-23-beta-location-analysis-phase-3.md:57]: We decided not to change the heading as it relates to the section.

File: app/posts/manage-school-placements/2024-02-23-beta-location-analysis-phase-3.md (hunk 2 above)
Comment (simonwhatley) [reply app/posts/manage-school-placements/2024-02-23-beta-location-analysis-phase-3.md:91]: We decided not to change the heading as it relates to the section.

File: app/posts/manage-school-placements/2024-02-23-beta-location-analysis-phase-3.md (hunk 3 above)
Comment (simonwhatley) [reply app/posts/manage-school-placements/2024-02-23-beta-location-analysis-phase-3.md:128]: We decided not to change the heading as it relates to the section.

File: app/posts/manage-school-placements/2024-02-23-beta-location-analysis-phase-4.md (hunk 4 above)
Comment (simonwhatley) [reply app/posts/manage-school-placements/2024-02-23-beta-location-analysis-phase-4.md:21]: We decided to change the numbers to bands so we don't show actual numbers. The bands are:

- 1 to 25
- 25 to 50
- 51 to 75
- 76 to 100
- over 100

**Pull request merged**